docs/routers/nokia-G240WB.md

```diff
@@ -0,0 +1,93 @@
+This guide was developed using a `Nokia G-240W-B` router (software version 3FE47969AGCA21). It should work for other models in the `G` series.
+
+!!! note
+    There is no single way to do it right. Choose the one best suited for you.
+This guide is `IPv4` and `IPv6` compatible, but it is not mandatory to use `IPv6`.
+
+### Prerequisites
+
+- The `Raspberry Pi` is using static IP address, `IPv4` and `IPv6` as well. This can be a fixed DHCP assignment or configured statically on the Pi itself.
+- The `ISP` provides `IPv6` address, prefix and gateway to the router.
+
+### Grab your IPv4 and IPv6 address from your Raspberry Pi
+
+Notice that the `IPv4` and `IPv6` addresses suffixes matches the number `xxx.168.1.10` and `xxxx::192:168:1:10` respectively.
+
+Worth to mention that the `IPv6` address that starts with `fe80` is the `link-local` address and it is used in this guide, the other addresses that start with `2806` are the global addresses and they are assigned by the router.
```

Comment:
This could be the last paragraph in "Grab your IPv4 and IPv6 address from your Raspberry Pi"